packages/preview/hydra/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+entrypoint = "src/lib.typ"
+name = "hydra"
+version = "0.1.0"
+compiler = "0.8.0"
+authors = ["tinger <me@tinger.dev>"]
+repository = "https://github.com/tingerrr/hydra"
+description = "Query and display headings of the currently active section"
+keywords = ["heading", "introspection"]
+license = "MIT"
+exclude = ["example", "LICENSE", "README.md"]
```

Comment:
The LICENSE should not be excluded. I also wouldn't exclude the README since it's small, but some packages do and it's not really important either way.

Reply:
I'll remove it from the exclusion, but can it even be accessed by the code in any meaningful way?

Reply:
It's not for technical reasons, it's just good practice to ship a LICENSE alongside code.